docs/usage/UsageWithTypescript.md

```diff
@@ -615,7 +615,7 @@ interface Book {
 }
 
 // highlight-next-line
-const booksAdapter = createEntityAdapter<Book>({
+const booksAdapter = createEntityAdapter<Book, number>({
   selectId: book => book.bookId,
   sortComparer: (a, b) => a.title.localeCompare(b.title)
 })
```

Comment:
```suggestion
const booksAdapter = createEntityAdapter({
  selectId: (book: Book) => book.bookId,
  sortComparer: (a, b) => a.title.localeCompare(b.title)
})
```
bigger change, but i think we should advise the version that's shorter and uses inference